go.mod

```diff
@@ -68,8 +69,9 @@ require (
 	github.com/NYTimes/gziphandler v1.1.1 // indirect
 	github.com/bronze1man/goStrongswanVici v0.0.0-20190828090544-27d02f80ba40 // indirect
 	github.com/containerd/cgroups v0.0.0-20200710171044-318312a37340
-	github.com/containerd/containerd v1.4.1
+	github.com/containerd/containerd v1.5.0-beta.0.0.20210122062454-5a66c2ae5cec
 	github.com/containerd/cri v1.11.1-0.20200820101445-b0cc07999aa5
+	github.com/containerd/stargz-snapshotter v0.4.2-0.20210215092109-c97b82620fb2
```

Comment:
Can we have a new release soon for this?

Reply:
Yes we have.